integration/modeshape-jbossas-integration-tests/pom.xml

```diff
@@ -231,6 +241,20 @@
                             </artifactItems>
                         </configuration>
                     </execution>
+                    
+                    <execution>
+                        <id>copy-dependencies</id>
+                        <phase>process-test-classes</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>                        
+                        <configuration>
```

Comment:
@hchiorean , in case of artifact-related configuration maven ignores transitive dependencies. Later, for WF module I need to specify jar names so for this particular case more useful to follow rule based on jar names. 

I did change dependency output directory name from generic `lib` to more specific `eslib`